src/calculateRentalCost.js

```diff
@@ -5,6 +5,28 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const daylyRent = 40;
```

Comment:
Consider renaming 'daylyRent' to 'DAILY_RENT' to indicate that it is a constant value and follow the naming convention for constants.